atmo/scheduling.py

```diff
@@ -98,3 +98,19 @@ def spark_job_run(user_email, identifier, notebook_uri, result_is_public, size,
         VisibleToAllUsers=True,
     )
     return cluster['JobFlowId']
+
+
+def get_spark_job_results(identifier, is_public):
+    if is_public:
+        bucket = settings.AWS_CONFIG['PUBLIC_DATA_BUCKET']
+    else:
+        bucket = settings.AWS_CONFIG['PRIVATE_DATA_BUCKET']
+    results = {'data': [], 'logs': []}
+    s3_objects = s3.list_objects_v2(Bucket=bucket, Prefix='{}/'.format(identifier))
+    for item in s3_objects.get('Contents', []):
+        try:
+            prefix = item['Key'].split('/')[1]
+        except IndexError:
+            continue
+        results[prefix].append(item['Key'])
```

Comment:
I forgot to mention that in the last review, in case this function finds a prefix that is not expected (e.g. because the notebook does something crazy and adds new folders to the data bucket) this will raise a `KeyError` since it can't find the item for the prefix in the results dict...since we pre-init it on line 108. This can be circumvented by doing the following:

First set results to an empty dict
```python
results = {}
```

Then use `setdefault` to make sure the item is there already
```python
results.setdefault(prefix, []).append(item['Key'])
```